examples/streaming/level_one_options.py

```diff
@@ -0,0 +1,214 @@
+import asyncio
+import logging
+import pprint
+from datetime import datetime, timedelta
+from typing import Callable, Optional
+from typing import List, Dict
+
+import schwab
+from schwab.client import AsyncClient
+from schwab.streaming import StreamClient
+
+API_KEY = "XXXXXX"
+CLIENT_SECRET = "XXXXXX"
+CALLBACK_URL = "https://xxxxxx"
+
+
+async def get_chains(client: AsyncClient, instruments: List[str]) -> Dict:
+    chains = {}
+
+    from_date = datetime.today().date()
+    to_date = from_date + timedelta(days=3)
+
+    for instrument in instruments:
+        chain = await client.get_option_chain(
```

Comment:
`await` needs to be removed from here....I was not able to get this running with `await`

Reply:
Did you use async client?

Reply:
I ran it in the vscode terminal.